core/src/mast/node/split_node.rs

```diff
@@ -16,7 +17,7 @@ pub struct SplitNode {
 /// Constants
 impl SplitNode {
     /// The domain of the split node (used for control block hashing).
-    pub const DOMAIN: Felt = Felt::new(Operation::Split.op_code() as u64);
+    pub const DOMAIN: Felt = Felt::new(OPCODE_SPLIT as u64);
```

Comment:
In this and other similar situations I would use `Felt::from(OPCODE_SPLIT)` as this would be a bit cleaner and modular reduction for smaller values is a tiny bit more efficient than from a full `u64` value.

Reply:
This (and others) are in a `const` context, and only `new()` is const. And so we don't really care about such small differences in performance, since it's done at compile-time